python-test/features/steps/local_agent.py

```diff
@@ -0,0 +1,90 @@
+from json import loads, JSONDecodeError
+from behave import when, then
+from hamcrest import *
+from test_config import TestConfig, LOCAL_AGENT_CONTAINER_NAME
+import docker
+
+configs = TestConfig.configs()
+
+
+@when('the agent container is started')
+def run_local_agent_container(context):
+    orb_address = configs.get('orb_address')
+    interface = configs.get('orb_agent_interface', 'mock')
+    image_tag = ':' + configs.get('agent_docker_tag', 'latest')
+    agent_image = "ns1labs/orb-agent" + image_tag
```

Comment:
Occurred to me we should make "ns1labs" a config variable as well - a community member may wish to use images from their own repository